recipes/bertopic/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "bertopic" %}
+{% set version = "0.9.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bertopic-{{ version }}.tar.gz
+  sha256: 7ffed1e7bb81e6b0f1039141809c9d157fa415d65c298e9887e70b890f274bce
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - pyyaml <6.0
```

Let's consider this temporary until upstream has fixed their issues. Could you please add a comment here so we don't forget? Thanks!

---

Do you mean we should exclude Windows for the time being? Thanks!

---

No, I meant the `pyyaml` version constrain (<6) should be considered a temporary workaround and if possible would be removed in the future when upstream has fixed their compatibility issues. Adding a comment saying `# This is a temporary workaround; review in the future` should be enough:
```suggestion
    # This is a temporary workaround; review in the future
    - pyyaml <6.0
```